src/core/lib/address_utils/parse_address.cc

```diff
@@ -103,8 +103,17 @@ grpc_error_handle UnixSockaddrPopulate(absl::string_view path,
         "Path name should not have more than ", maxlen, " characters"));
   }
   un->sun_family = AF_UNIX;
-  path.copy(un->sun_path, path.size());
-  un->sun_path[path.size()] = '\0';
+  int offset_start = 0;
+  if (absl::StartsWith(path, "///")) {
```

Comment:
This could potentially be cleaner with 
```
if (absl::ConsumePrefix(&path, "//") && !absl::StartsWith(path, "/")) {
  return /* error */;
}
path.copy(un->sun_path, path.size());
...
```

Reply:
Theses debugging output should not have been left over, i'll purge them

Reply:
See https://github.com/abseil/abseil-cpp/blob/0941ce7fd95104ba5f4678c41802a3229db1d5eb/absl/strings/strip.h#L36-L40. The only reject path here requires the string to start with `//`